### PR TITLE
Update DB versions in docker demos

### PR DESCRIPTION
**Pull request by shadinua**
* PostgreSQL 9.6 -> 11
* MySQL 5.7 -> MariaDB 10.3

### CODE REVIEW

**Comment by vixentael** [comment]
yey, MariaDB in da house!

File: examples/python/example_with_zone.py [hunk 1]
```diff
@@ -141,14 +143,14 @@ def write_data(data, connection):
     metadata.create_all(engine)
 
     if args.print:
-        print_data(args.zone_id, connection)
+        print_data(args.zone_id, connection, driver)
```
Comment (Lagovas) [inline examples/python/example_with_zone.py]: what about place `print('DB driver: {}'.format(driver))` before `if args.print else ` statement and don't pass it to 2 functions with extra parameter?

File: examples/python/example_without_zone.py [hunk 2]
```diff
@@ -109,6 +111,6 @@ def write_data(data, connection):
     connection = engine.connect()
 
     if args.print:
-        print_data(connection)
+        print_data(connection, driver)
```
Comment (Lagovas) [inline examples/python/example_without_zone.py]: same here

File: examples/python/example_with_zone.py (hunk 1 above)
Comment (shadinua) [reply examples/python/example_with_zone.py]: Sure, thanks :-)

**Pull request merged**